app/models/validators/range_overlapping_validator.rb

```diff
@@ -1,5 +1,7 @@
 class RangeOverlappingValidator < ActiveModel::Validator
   def validate(record)
+    return if record.metric_configuration.nil?
```

Comment:
What do you think about validating this on the model side? (https://github.com/mezuro/kalibro_configurations/blob/master/app/models/kalibro_range.rb#L12)

Is there a test to replicate the problem handled here?